config/configuration.toml.in

```diff
@@ -1,6 +1,6 @@
 # XXX: Warning: this file is auto-generated from file "@CONFIG_IN@".
 
-[hypervisor.qemu-lite]
+[hypervisor.q35]
```

Comment:
My two rusty cents: ideally it would be nice to support both: current upstream QEMU and future version with new shiny features. Runtime then could dynamically detect the QEMU version and enable supported features when it launches a new VM. 
I understand that maintaining cost might slightly increase, but this approach could enable CC on platforms with outdated QEMU.
Just an opinion.